rest/taskana-rest-spring-test/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java

```
@@ -282,26 +278,28 @@ public void testGetClassificationWithSpecialCharacter() {
         assertEquals("Zustimmungserklärung", response.getBody().name);
     }
 
-    @Test(expected = HttpClientErrorException.class)
+    @Test
     @DirtiesContext
-    public void testDeleteClassification() {
+    void testDeleteClassification() {
         RestTemplate template = getRestTemplate();
         HttpHeaders headers = new HttpHeaders();
         headers.add("Authorization", "Basic dGVhbWxlYWRfMTp0ZWFtbGVhZF8x");
         HttpEntity<String> request = new HttpEntity<String>(headers);
 
-        ResponseEntity<ClassificationSummaryResource> response = template.exchange(
-            "http://127.0.0.1:" + port + "/api/v1/classifications/CLI:200000000000000000000000000000000004",
-            HttpMethod.DELETE,
-            request,
-            ParameterizedTypeReference.forType(ClassificationSummaryResource.class));
-        assertEquals(HttpStatus.NO_CONTENT, response.getStatusCode());
-
-        response = template.exchange(
-            "http://127.0.0.1:" + port + "/api/v1/classifications/CLI:200000000000000000000000000000000004",
-            HttpMethod.GET,
-            request,
-            ParameterizedTypeReference.forType(ClassificationSummaryResource.class));
+        assertThrows(HttpClientErrorException.class, () -> {
+            ResponseEntity<ClassificationSummaryResource> response = template.exchange(
```

which call should throw the exception?

---

fixed and done. was the 2nd call.